conf/battle/drops.conf

```diff
@@ -206,3 +206,8 @@ drop_rate_cap_vip: 9000
 
 // Displays a colored pillar effect for items dropped by monsters that contain random options.
 rndopt_drop_pillar: on
+
+// Drop rate bonus can increase drop rate from map_drops.
+// On official servers, level difference penalties do not affect map drops.
+// Default: on
+enable_bonus_map_drops: on
```

Comment:
I'm not sure if that's actually true. Drop rate modifiers might actually work for map drops officially. I'm not sure if there's an easy way to test this, though.